src/mpid/ch4/generic/am/mpidig_am.h

```diff
@@ -9,13 +9,11 @@
 #define MPIDI_AM_HANDLERS_MAX (64)
 
 enum {
-    MPIDIG_SEND = 0,            /* Eager send */
+    MPIDIG_SEND = 0,
 
-    MPIDIG_SEND_LONG_REQ,       /* Rendezvous send RTS (request to send) */
-    MPIDIG_SEND_LONG_ACK,       /* Rendezvous send CTS (clear to send) */
-    MPIDIG_SEND_LONG_LMT,       /* Rendezvous send LMT */
+    MPIDIG_DATA_CTS,    /* general CTS (clear to send) */
+    MPIDIG_DATA_SEG,    /* general segmented data */
```

Comment:
Just to make it clear --
*  `MPIDIG_SEND` now handles both original `MPIDIG_SEND` and `MPDIG_SEND_LONG_REQ`; 
*  `MPIDIG_DATA_CTS` replaces the old `MPIDIG_SEND_LONG_ACK`
*  `MPIDIG_DATA_SEG` replaces the old `MPIDIG_SEND_LONG_LMT`
right?

If so, I think `MPIDIG_SEND_CTS` and `MPIDIG_SEND_DATA` are more intuitive names.

Reply:
> Just to make it clear --
> 
> * `MPIDIG_SEND` now handles both original `MPIDIG_SEND` and `MPDIG_SEND_LONG_REQ`;
> * `MPIDIG_DATA_CTS` replaces the old `MPIDIG_SEND_LONG_ACK`
> * `MPIDIG_DATA_SEG` replaces the old `MPIDIG_SEND_LONG_LMT`
>   right?
That's right.


> If so, I think `MPIDIG_SEND_CTS` and `MPIDIG_SEND_DATA` are more intuitive names.
I think a general name is better as the CTS and SEG would be used to handle more than just SEND. For example, fallback when ZCOPY is rejected.

Reply:
What bothers me is `MPIDIG_DATA_SEG`, because the "segment" is handled inside netmod/shmmod and not visible at the ch4-layer.

Reply:
In addition, I am bothered by the names being changed too radically that folks trying to integrate may get lost on what is going on. Because we didn't change the functionality at all here, I suggest make name changes subtle to be less confusing.